src/util/compose.js

```diff
@@ -56,10 +60,23 @@ services:
       JWT_TOKEN_EXPIRES: 15
     env_file:
       - ../{{ env_file }}
+  api:
+    container_name: nhost_api
+    build:
+      context: ../../
+      dockerfile: nhost/.nhost/Dockerfile-api
+    environment:
+      PORT: 4000
+    ports:
+      - "{{ api_port }}:4000"
```

Comment:
to be consistent with the other port definitions, could we use `api_port` on all references to `4000`?. I think it is fine to assume the same port mapping between the host and the container.